src/PortingAssistant.Client.Analysis/Utils/PackageCompatibility.cs

```diff
@@ -122,6 +122,22 @@ public static async Task<CompatibilityResult> IsCompatibleAsync(Task<PackageDeta
                     CompatibleVersions = new List<string>()
                 };
             }
+
+            catch (AggregateException ae)
+            {
+                foreach (var e in ae.Flatten().InnerExceptions)
+                {
+                    if (!(e is PortingAssistantClientException))
```

Comment:
why we need to make this check?

Reply:
yeah, removed the check and remove the error log in previous caller.  so it will only have one exception logged for each package not found issue.